deploy/kicbase/containerd.toml

```diff
@@ -57,9 +57,8 @@ oom_score = 0
       conf_dir = "/etc/cni/net.mk"
       conf_template = ""
     [plugins."io.containerd.grpc.v1.cri".registry]
-      [plugins."io.containerd.grpc.v1.cri".registry.mirrors]
-        [plugins."io.containerd.grpc.v1.cri".registry.mirrors."docker.io"]
```

Comment:
Would you please clarify, Why these options were removed ?

Reply:
@medyagh The configuration structure that containerd expects has changed with this new version. This entry is recreated through a directory structure instead now.

https://github.com/containerd/containerd/blob/main/docs/hosts.md